lib/is_perfect.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var divisors = require('./divisors');
+
+/**
+  * Determines whether an integer is 'perfect'.
+  * @param {Number} an integer x to test
+  * @return {Boolean} Whether or not x is perfect.
+  * @module number-theory
+  * @author Kelly Innes
+  */
+
+module.exports = function isDeficient(n) {
+  var divisorsOfNumber = divisors(n);
```

Comment:
Is there a faster algorithm known for this? I kinda doubt it but divisors is expensive :( worth checking out if you haven't already